data/data/agent/systemd/units/agent-auth-token-status.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=service that displays a message if agent auth token is expired.
+Wants=network-online.target
+After=network-online.target agent-interactive-console.service
+ConditionPathExists=/etc/assisted/add-nodes.env
+
+[Service]
+Type=simple
+EnvironmentFile=/etc/assisted/add-nodes.env
+ExecStart=/usr/local/bin/agent-auth-token-status.sh
+Restart=no
+
+[Install]
+WantedBy=multi-user.target agent-add-node.service
```

Comment:
Having `WantedBy: multi-user.target` means this will get enabled on the ISO even for day 1 installs (where there is no expiry time for the token). Will that break anything?

Reply:
The pre-submits were fine. The service has `ConditionPathExists=/etc/assisted/add-nodes.env` which ensures it will only start on day2

Reply:
I could remove `multi-user.target`

Reply:
Ah, OK. This is probably something we can clean up in the future to make it more obvious what's going on, but the Condition makes it safe for now so no need to change this PR.

Reply:
I handled this in #8804